app/views/users/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div id='clearance' class='sign-up'>
+  <h2><%= t('.title') %></h2>
+
+  <%= form_for @user do |form| %>
+  <div class='text-field'>
+    <%= form.label :name %>
+    <%= form.text_field :name, :type => 'name' %>
+  </div>
+
+  <div class='text-field'>
+    <%= form.label :email %>
+    <%= form.text_field :email, :type => 'email' %>
+  </div>
+
+  <div class='password-field'>
+    <%= form.label :password %>
+    <%= form.password_field :password %>
+  </div>
+
+  <div class='submit-field'>
+    <%= form.submit %>
+  </div>
+
+  <div class='other-links'>
+    <%= link_to t('.sign_in'), sign_in_path %>
+  </div>
+  <% end %>
```

Comment:
How very rude to correctly assume the code comes from elsewhere ;) Now indented.